CONTRIBUTE.md

```diff
@@ -0,0 +1,141 @@
+# Contributing
+
+The `go-python` project (and `gopy`) eagerly accepts contributions from the community.
+
+## Introduction
+
+
+The `go-python` project provides libraries and tools in Go for the Go community to better mesh with Python projects and libraries, and we would like you to join us in improving `go-python`'s quality and scope.
+This document is for anyone who is contributing or interested in contributing.
+Questions about `go-python` or the use of its libraries can be directed to the [go-python](mailto:go-python@googlegroups.com) mailing list.
+
+## Contributing
+
+### Working Together
+
+When contributing or otherwise participating, please:
+
+- Be friendly and welcoming
+- Be patient
+- Be thoughtful
+- Be respectful
+- Be charitable
+- Avoid destructive behavior
+
+Excerpted from the [Go conduct document](https://golang.org/conduct).
+
+### Reporting Bugs
+
+When you encounter a bug, please open an issue on the corresponding repository.
+Start the issue title with the repository/sub-repository name, like `bind: issue name`.
+Be specific about the environment you encountered the bug in (_e.g.:_ operating system, Go compiler version, ...).
+If you are able to write a test that reproduces the bug, please include it in the issue.
```

Comment:
I don't like the dual use of "test" in this bullet and the next. It's likely to be somewhat confusing for newer users.

"If you are able to write a reproducer for the bug, please include it in the issue"